src/main/java/uk/gov/justice/digital/hmpps/csr/api/model/Detail.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.csr.api.model
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class Detail(
+
+        val quantumId: String?,
+
+        val shiftModified: LocalDateTime?,
```

Comment:
shiftModified or detailModified?

Reply:
This value does actually seem to be shift modified.
https://github.com/ministryofjustice/cmd-csr-procs/blob/master/Notification_Shift_Detail/GET_MODIFIED_DETAIL.sql#L51
There is something in those comments that I think we need to talk to Mons about it, they're talking about including 'deleted' rows but in the code they're hardcoding all details to 'EDIT'